src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+$gap: 1px;
+$day-height: 100px;
+$day-width: 100px;
+$days-week: (1 'mon', 2 'tue', 3 'wed', 4 'thu', 5 'fri', 6 'sat', 7 'sun');
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Try don't use * selector (it impacts performance) better use html or body tag.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/296cfb57-b44e-44a4-9aae-8ee39ffd7a87)